src/main/kotlin/com/hibob/feedback/dao/modles.kt

```diff
@@ -0,0 +1,37 @@
+package com.hibob.feedback.dao
+
+import java.util.*
+
+data class Feedback(
+    val id: UUID,
+    val creationDate: Date,
+    val department: Department,
+    val companyId: UUID,
+    val status: Status,
+    val responseIdList: List<UUID>,
```

Comment:
i don't think that we need this. we will have a dedicated api for the responses in phase 2, let's leave this off for now